groups.json

```diff
@@ -1,12 +1,42 @@
 [
   {
-    "name": "Greene Lab",
-    "description": "Where data and biology meet",
-    "link": "https://greenelab.com"
+    "name": "Gignoux Lab",
+    "description": "At the interface of statistical and population genetics",
+    "link": "https://www.gignouxlab.org"
+  },
+  {
+    "name": "Way Lab",
+    "description": "Biomedical data science and software for translational research",
```

Comment:
```suggestion
    "description": "Biomedical data science and software for drug discovery and translational research",
```

Reply:
The text here needs to be less than about ~80 characters because it needs to fit into a tooltip. Also the text version is for printing.